package.json

```diff
@@ -45,6 +46,7 @@
     "express": "3.4.8",
     "grunt": "0.4.2",
     "grunt-browserify": "1.3.2",
+    "grunt-cli": "1.2.0",
```

Comment:
Adding this means that you don't need to have grunt globally installed. The npm scripts will use the local version of grunt to run the builds.